test-utils/testlib/src/fees_utils.rs

```diff
@@ -4,7 +4,7 @@ use near::config::INITIAL_GAS_PRICE;
 use near_primitives::types::{Balance, Gas};
 use near_runtime_fees::RuntimeFeesConfig;
 
-const GAS_PRICE: u128 = INITIAL_GAS_PRICE;
+pub const GAS_PRICE: u128 = INITIAL_GAS_PRICE;
```

Comment:
why do we need this constant if it's just one from near::config?

Reply:
Runtime didn't import config directly, but use testlib fees. I can import near::config as well.